app/resources/translations/cs_CZ/messages.cs_CZ.yml

```diff
@@ -7,75 +7,411 @@
 #          process and your PR so that it will eventually be necessary to
 #          re-map your translations.
 
-#   0 Unused messages
-#   8 Legacy untranslated messages
-#  18 Untranslated messages
-#  32 Legacy translation messages
-# 540 Translation messages
+# 361 Unused messages
+#   5 Legacy untranslated messages
+#   8 Untranslated messages
+#  20 Legacy translation messages
+# 197 Translation messages
+
+#--- Unused messages ----------------------------------------------------------
+
+"(default template)": "(standardní šablona)"
+"(no category)": "(bez kategorie)"
+"(no group)": "(nezařazení do skupiny)"
+"(none)": "(nic)"
+"Add some sample <a href='%url%' class='btn btn-default'>Records with Loripsum text</a>": "Přidat ukázku <a href='%url%' class='btn btn-default'>Záznam s Lorem ipsum</a>"
+"Current (saved) status:": "Současný (uložený) stav:"
+"Do you really want to delete %FOLDERNAME%?": "Opravdu smazat %FOLDERNAME%?"
+"Excerpt": "Souhrn"
+"Expand sidebar": "Roztáhnout postranní lištu"
+"The settings for roles and permissions are stored in <code>%permissions%</code>.": "Nastavení uživatelský práv a rolí je v <code>%permissions%</code>."
+"Title": "Nadpis"
+"View (saved version) on site": "Zobrazit uloženou verzi na webu"
+"contenttypes.generic.actions-all": "Akce pro %contenttypes%"
+"contenttypes.generic.actions-one": "Akce pro %contenttype%"
```

Comment:
Dotted keywords shouldn't be quoted. Is this coming out of the tool you're using?

Reply:
Dotted keywords? I didn't change the strings. I can see in the github or in the download those:

```
"You've been logged on successfully.": "You've been logged on successfully."
contenttypes.generic.actions-all: "Actions for %contenttypes%"
```

Reply:
@GawainLynch it was not me, but the Bolt lint tool. All "Unused messages" messages gets the keyword quoted after been passed by the language lint tool from Bolt (Admin / config / messages)

Example: Strings in the downloaded file for cs_CZ:

```
"Title": "Nadpis"
"Everybody": "Všichni"
users.your-password: "Vaše heslo"
contenttypes.generic.group.taxonomy: "Taxonomie"
contenttypes.generic.group.template: "Šablona"

```

Strings after been saved by BOLT:
```
#--- Unused messages ----------------------------------------------------------

"Title": "Nadpis"
"users.your-password": "Vaše heslo"


#--- Legacy translation messages ----------------------------------------------

"Everybody": "Všichni"

#--- Translation messages -----------------------------------------------------

contenttypes.generic.group.taxonomy: "Taxonomie"
contenttypes.generic.group.template: "Šablona"

```
As you can see the Unused and Legacy messages are quoted by Bolt. The "translations section are not edited by BOLT.
So this shouldn't be a problem to block this PR but a bug alert to solve in a different PR.